.travis.yml

```diff
@@ -67,9 +60,7 @@ matrix:
 
     - os: linux
       env: NAME='full'
-           NOSE_COVERAGE='--with-coverage --cover-package MDAnalysis'
-           PYTEST_COVERAGE='--cov=MDAnalysis'
-           MAIN_CMD='export COVERAGE_FILE=$PYTEST_COVERAGE_FILE; pytest ${PYTEST_LIST} ${PYTEST_FLAGS} ${PYTEST_COVERAGE}; export COVERAGE_FILE=$NOSE_COVERAGE_FILE; python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_TEST_LIST} ${NOSE_FLAGS} ${NOSE_COVERAGE}'
+           MAIN_CMD='pytest testsuite/MDAnalysisTests/ ${PYTEST_FLAGS} --cov=MDAnalysis'
```

Comment:
can't you just update the `PYTEST_FLAGS` variable here instead of rewriting `MAIN_CMD`?